src/data/consts.js

```diff
@@ -1,2 +1,5 @@
 //skapps paths to load posts from
 export const skynetBlogHNS = ["skynetblog.hns"];
+
+//domain at which to save json
+export const dataDomain = "skynetblog.hns";
```

Comment:
REQ: these should be combined. Probably dataDomain is the right const. Then where you are using `skynetBlogHNS` just do `[dataDomain]` instead of the `[ ]` being part of the const.